### PR TITLE
SED-1986 When scrolling tables global actions should stay fixed

### PR DESCRIPTION
**Pull request by bojidaryovchev**

### CODE REVIEW

File: projects/step-core/src/lib/modules/table/components/table/table.component.scss [hunk 1]
```diff
@@ -1,10 +1,30 @@
-:host {
-  position: relative;
-  display: block;
+step-table {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
```
Comment (dvladir) [inline projects/step-core/src/lib/modules/table/components/table/table.component.scss:5]: According to the tasks description - the issue was about controls which are displayed above the table.
On the first glance, the requested behavior can be achieved just by adding a single line in the `host` rule
```
:host {
   position: relative;
   display: block;
  
   // this line is need to be added
   min-width: fit-content;
}
```

What benefits PR's approach gives in comparison with solution, that is shown above?

File: projects/step-core/src/lib/modules/table/components/table/table.component.scss [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+    flex: 1;
+    overflow: auto;
+    display: block;
+
+    table {
+      width: 100%;
+      border-collapse: separate;
+
+      thead tr {
+        position: sticky;
```
Comment (dvladir) [inline projects/step-core/src/lib/modules/table/components/table/table.component.scss:16]: There is no information in the ticket's description, that sticky headers should be implemented. Why it was added?

Comment (bojidaryovchev) [reply projects/step-core/src/lib/modules/table/components/table/table.component.scss:16]: better UX

Comment (neogucky) [reply projects/step-core/src/lib/modules/table/components/table/table.component.scss:16]: I actually like it. We just have to be careful in places where there could be multiple tables, but as far as I saw all works well!

**Review by neogucky — APPROVED** [review]

**Pull request merged**